angular/core/components/dashboard_page/dashboard_page.scss

```diff
@@ -55,3 +57,14 @@
 .dashboard-page__show-more a:hover{
   color: $primary-text-color;
 }
+
+.dashboard-page__mute-image--large {
+  img { max-width: 730px; }
```

Comment:
Where does this number come from? Could we get away with $small-max (768px) instead of adding an arbitrary value, perhaps adding margins to the img if need be?